includes/class-sensei-course.php

```diff
@@ -4195,6 +4197,45 @@ public function archive_page_content() {
 			remove_all_actions( 'sensei_pagination' );
 		}
 	}
+
+	/**
+	 * Take the user to login page if trying to access course results page without being logged in.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @access private
+	 *
+	 * @return void
```

Comment:
An FYI that we don't need to include `@return void` for functions without a return value as per https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods.